ui/partials/main/thShortcutTable.html

```diff
@@ -4,7 +4,7 @@
       <a ng-show="onscreenShortcutsShowing"
          ng-click="setOnscreenShortcutsShowing(false)"
          class="pull-right" href="" title="Close this help">
-        <span class="fa fa-times lightgray"</span>
+        <span class="fa fa-times lightgray"></span>
```

Comment:
This will not be in the PR once I rebase over @edmorley's PR1014, which will get landed first.